views/js/pciCreator/ims/likertScaleInteraction/creator/tpl/markup.tpl

```diff
@@ -1,4 +1,4 @@
-<div class="likertScaleInteraction">
+<div class="likertInteraction">
```

Comment:
This identifier is already taken by `pciCreator/ims/likertConfig`. (That's _another_ version of the PCI, which never had an install script. I have no idea which environments it was ever added to. Also its version is 1.2.0)

If using this PR, we should either choose a brand new identifier, or delete `likertConfig`.